l10n_ro_report_trial_balance/report/trial_balance.py

```diff
@@ -0,0 +1,261 @@
+    create_date,
+    account_id,
+    code,
+    name,
+    debit_opening,
+    credit_opening,
+    debit_initial,
+    credit_initial,
+    debit,
+    credit,
+    debit_total,
+    credit_total,
+    debit_balance,
+    credit_balance)
+SELECT
+    %s AS report_id,
+    %s AS create_uid,
+    NOW() AS create_date,
+    accounts.*,
+    CASE WHEN accounts.debit_total > accounts.credit_total
+        THEN accounts.debit_total - accounts.credit_total
+        ELSE 0
+    END AS debit_balance,
+    CASE WHEN accounts.credit_total > accounts.debit_total
+        THEN accounts.credit_total - accounts.debit_total
+        ELSE 0
+    END AS credit_balance
+FROM
+    (
+    SELECT
+        acc.id, acc.code, acc.name,
+        coalesce(sum(open.debit),0) AS debit_opening,
+        coalesce(sum(open.credit),0) AS credit_opening,
+        coalesce(sum(init.debit),0) AS debit_initial,
+        coalesce(sum(init.credit),0) AS credit_initial,
+        coalesce(sum(current.debit),0) AS debit,
+        coalesce(sum(current.credit),0) AS credit,
+        coalesce(sum(open.debit),0) + coalesce(sum(init.debit),0) +
+            coalesce(sum(current.debit),0) AS debit_total,
+        coalesce(sum(open.credit),0) + coalesce(sum(init.credit),0) +
+            coalesce(sum(current.credit),0) AS credit_total
+    FROM
+        account_account acc
+        LEFT OUTER JOIN account_move_line AS open
+            ON open.account_id = acc.id AND open.date < %s
+        LEFT OUTER JOIN account_move_line AS init
+            ON init.account_id = acc.id AND init.date >= %s AND init.date < %s
+        LEFT OUTER JOIN account_move_line AS current
+            ON current.account_id = acc.id AND current.date >= %s
+                AND current.date <= %s
+        LEFT JOIN account_move AS move
+            ON open.move_id = move.id AND move.state in (%s)
+        LEFT JOIN account_move AS init_move
+            ON init.move_id = init_move.id AND init_move.state in (%s)
+        LEFT JOIN account_move AS current_move
+            ON current.move_id = current_move.id AND current_move.state in (%s)
+    WHERE acc.id in %s
+    GROUP BY acc.id
+    ORDER BY acc.code) as accounts"""
+        query_inject_account_params = (
+            self.id,
+            self.env.uid,
+            fy_start_date,
+            fy_start_date, self.date_from,
+            self.date_from, self.date_to,
+            states,
+            states,
+            states,
+            accounts._ids,
+        )
+        self.env.cr.execute(query_inject_account, query_inject_account_params)
+
+    def _compute_account_group_values(self):
+        if not self.account_ids:
+            acc_res = self.line_account_ids
+            groups = self.env['account.group'].search(
+                [('code_prefix', '!=', False)])
+            for group in groups:
+                accounts = acc_res.filtered(
+                    lambda a: a.account_id.id in
+                    group.compute_account_ids.ids)
```

Comment:
compute_account_ids  nu exista in account.group

Reply:
Yep, pentru widget, compute_account_ids exista aici, https://github.com/feketemihai/account-financial-reporting/blob/e38193495b18e4e9f52112b083ae2286cc71cb98/account_financial_report/models/account_group.py#L22, si modulul depinde de account_financial_report ca sa nu adaug de 2 ori campu lrespectiv. Cel mai bine ar fii sa facem review in Engleza atat timp cat suntem putini, si o sa depinde de Maintainers..